src/fetch.py

```diff
@@ -37,14 +43,14 @@ def get_source(self):
         """
 
         try:
-            response = self.session.get(self.swagger_url)
+            response = self.session.get(self.swagger_url, headers ={'X-XSRF-Header': '9999999'})
```

Comment:
can you add a comment here to explain the 9999999

Reply:
Ping Access requires an `X-XSRF-Header` HTTP request header to be included to protect against cross-site request forgeries. The value of this header can be set to anything so long as it is included in the request. We set it to a static 9999999 here but would use a randomly generated value in automation scripts.